recipes/jupyterlab-server/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: jupyterlab_server
+  version: {{ version }}
+
+source:
+  fn: jupyterlab_server-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/j/jupyterlab_server/jupyterlab_server-{{ version }}.tar.gz
+  sha256: a5373cda6579ca4504b38068a0cb4eba3b72b032a0de44291062788abf6d0f43
+
+build:
+    number: 0
+    noarch: python
+    script: python -m pip install --no-deps  --ignore-installed .
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - setuptools
+
+  run:
+    - python >=3.5
+    - notebook
+
+test:
+  imports:
+    - jupyterlab_server
+
+about:
+  home: http://github.com/jupyterlab/jupyterlab_server
+  license: BSD 3-Clause
+  license_file: LICENSE
+  summary: A set of server components for JupyterLab and JupyterLab like applications.
+
+extra:
+  recipe-maintainers:
+    - jasongrout
+    - blink1073
+    - afshin
+    - SylvainCorlay
```

Comment:
all of these people already agreed to be a maintainer of this recipe?

@blink1073 , @afshin , @SylvainCorlay

Reply:
👍 from me

Reply:
👍

Reply:
cc @jasongrout

Reply:
Yep!